modules/preview-react/table/stories/examples/BasicWithHeading.tsx

```diff
@@ -6,7 +6,9 @@ import {Heading} from '@workday/canvas-kit-react/text';
 export const BasicWithHeading = () => {
   return (
     <>
-      <Heading size="small">Pizza Toppings</Heading>
+      <Heading size="small" marginBottom="s">
```

Comment:
Added bottom margins to all example `Heading`s to ensure the margins show up even when a CSS reset is used.

Reply:
Perfect